wgsl/index.bs

```diff
@@ -2758,12 +2781,9 @@ A type is <dfn noexport>host-shareable</dfn> if it is both [=type/concrete=] and
 * a [=runtime-sized=] array type, if its element type is host-shareable
 * a [=structure=] type, if all its members are host-shareable
 
-Note: Many types are host-shareable, but not IO-shareable, including [=atomic types=],
-[=runtime-sized=] arrays, and any composite types containing them.
-
-Note: Both IO-shareable and host-shareable types have specified sizes, but counted differently.
```

Comment:
Nice, thank you. Removing IO-shareable is good since it's nowhere else, as I recall.